css/src/components/tabs.scss

```diff
@@ -0,0 +1,98 @@
+$tabs-padding: $font-size-9 $font-size-5 !default;
+$tabs-padding-lg: $font-size-7 $font-size-4 !default;
+
+$tab-font-color: $primary !default;
+$tab-border: $border-width solid transparent !default;
+$tabs-border: $border-width solid $border !default;
+$tabs-hover-background-color: $primary-background !default;
+$tabs-hover-border: $border-width solid $border !default;
+$tabs-hover-border-color: $primary-hover !default;
+$tabs-active-hover-background-color: $primary-hover !default;
+$tabs-active-hover-border-color: $primary-hover !default;
+$tabs-active-background-color: $primary-active;
+$tabs-active-border-color: $primary-active !default;
+$tabs-active-color: $text-invert !default;
+
+[role='navigation'] {
```

Comment:
We want to keep the previous classes around for styling purposes. we can leave the web component to target accessibility attributes

Reply:
updated